test/jdk/javax/swing/JButton/TimeChangeButtonClickTest.java

```diff
@@ -28,193 +28,99 @@
  * decreasing system's time
  * @run main/manual TimeChangeButtonClickTest
```

Comment:
Please add jtreg tags to avoid compilation error.
`@library /java/awt/regtesthelpers` and `@build PassFailJFrame`

Reply:
Good catch! I forgot to add them.